.gitignore

```diff
@@ -6,9 +6,13 @@ tmp
 .DS_Store
 .idea
 public/build/
-npm-debug.log
+npm-debug.log*
 config/*.local.*
 public/*
 !public/index.html
```

Comment:
@nacho87 can we clean up this I think we can use something like `!public/*.html`